README.md

```diff
@@ -4,29 +4,84 @@
 
 The fastest way to install [Micro](https://micro-editor.github.io/)
 
-`curl https://getmic.ro | bash`
+```Bash
+# local install just for your user
+curl https://getmic.ro | sh
+```
 
-This script will install micro to the directory you're in. To install somewhere else (e.g. /usr/local/bin), cd there and make sure you can write to that directory, e.g. `cd /usr/local/bin; curl https://getmic.ro | sudo bash`
+Or, using wget in place of curl:
 
-> Vous ne comprenez pas l'anglais? vous parlez français? Regardez le [*LISEZ-MOI* français](./README.fr.md)!
+```Bash
+# local install just for your user
+wget -O- https://getmic.ro | sh
+```
 
-## Advanced usage
+This script will install micro to the directory you're in. To install somewhere else (e.g. /usr/local/bin), cd there and make sure you can write to that directory, e.g. `cd /usr/local/bin; curl https://getmic.ro | sudo sh` like so:
+
+```Bash
+# global install for all users
+cd /usr/bin
+curl https://getmic.ro | sudo sh
+```
+
+Or, using wget and su in place of curl and sudo:
 
-There's a couple other things you can do with getmic.ro.
+```Bash
+# global install for all users
+su - root -c 'cd /usr/bin; wget -O- https://getmic.ro sh'
+```
 
-### Use another POSIX shell
+> Vous ne comprenez pas l'anglais? vous parlez français? **Regardez le [*LISEZ-MOI* français](./README.fr.md)!**
 
-While getmic.ro is tested primarily with bash, it should be compatible with any POSIX shell (we test zsh, dash, ksh, and busybox). If you don't have `bash`, you can just use:
+## Advanced usage
 
-`curl https://getmic.ro | sh`
+There's a couple other things you can do with getmic.ro. Listed below are environment variables you can choose from:
+
+* `GETMICRO_HTTP=<COMMAND ...ARGS>`
+    + Use this command with arguments deliminated by spaces to download files off the internet.
+        - It MUST follow redirects (via HTTP Location headers).
+        - It MUST print the resulting file contents to stdout.
+        - It MUST accept the next argument to be the URL of the file to be downloaded
+        - It IS OPTIONAL for the command to also accept a `--header` parameter used for non-essential GitHub authentication fallback shim.
+    + For example, to force using `curl`, do: `curl https://getmic.ro | GETMICRO_HTTP="curl -L" sh`
+    + For example, to force using `curl`, do: `wget -O- https://getmic.ro | GETMICRO_HTTP="wget -O-" sh`
```

Comment:
Should this read ``For example, to force using `wget` ``?

Reply:
Yes. My bad. Thank you for catching this.